requirements_dev.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+bumpversion
```

Comment:
Are you planning on using `bumpversion`?

Reply:
Oh, I was under the impression that it would be needed to make new versions on a new tag. Looking at `pypi-publish.yml`, I guess it's not necessary.

Reply:
I think it is something they use in the `birdhouse` projects but I'm not familiar with it.